plugin.video.nhlgcl/addon.xml

```diff
@@ -1,23 +1,24 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="plugin.video.nhlgcl" name="NHL TV™" version="2016.11.18" provider-name="eracknaphobia">
+<addon id="plugin.video.nhlgcl" name="NHL TV™" version="2017.1.24" provider-name="eracknaphobia">
   <requires>
    <import addon="xbmc.python" version="2.19.0"/>   
```

Comment:
When you have some time (next update) please consider setting the xbmc.python version to 2.24.0  so it matches jarvis